transports/webrtc-websys/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
   See [PR 4959](https://github.com/libp2p/rust-libp2p/pull/4959).
 - Remove `libp2p_noise` from the public API.
   See [PR 4969](https://github.com/libp2p/rust-libp2p/pull/4969).
+- Bump version of web-sys and update `__Nonexhaustive` to `__Invalid`
+  See [PR 5569](https://github.com/libp2p/rust-libp2p/pull/5569)   
```

Comment:
We should move this under `0.4.0-alpha.2` since libp2p-webrtc-websys is released as `0.4.0-alpha` (you might need to pull the main branch) :)